docs: add troubleshooting (#208)
headspinio/appium-tizen-tv-driver: packages/appium-tizen-tv-driver/README.md

npm run watch
```

## Troubleshooting

> - The application under tests did not start by `sdb -s <device> shell 0 debug <package id>' exited with code 1`

Reviewer: ```suggestion
- The application under test did not start; `sdb -s <device> shell 0 debug <package id>' exited with code 1`
```

>     - Please make sure if the application under tests id debuggable

Reviewer: ```suggestion
    - Please make sure the application under test is debuggable
```
